app/src/main/java/com/sopt/now/compose/LoginViewModel.kt

```diff
@@ -0,0 +1,64 @@
+import android.content.Context
+import android.content.Intent
+import android.widget.Toast
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.sopt.now.compose.MainActivity
+import com.sopt.now.compose.api.ServicePool
+import com.sopt.now.compose.dto.RequestLogInDto
+import com.sopt.now.compose.dto.ResponseLogInDto
+import kotlinx.coroutines.launch
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class LoginViewModel : ViewModel() {
+
+    var loginId = mutableStateOf("")
+    var loginPw = mutableStateOf("")
```

Comment:
recomposition을 위해 이렇게 바꾸는건 어떨까요?
```suggestion
    var loginId by remember { mutableStateOf("") }
    var loginPw by remember { mutableStateOf("") }
```